programs/server/embedded.xml

```diff
@@ -12,7 +12,6 @@
 
     <path>./</path>
 
-    <uncompressed_cache_size>8589934592</uncompressed_cache_size>
```

Comment:
This is not related to #32493. Is it an intentional change?

Reply:
Yes, just in case.